### PR TITLE
Initial work on load tests

### PR DESCRIPTION
**Pull request by pdehaan**
Just a copy of @nhnt11's load tests from https://pastebin.mozilla.org/9087582
Putting into source control for easier tracking and collaboration.

### CODE REVIEW

File: loadtests/loadtest.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from molotov import scenario
+
+
+HOMEPAGE = 'https://fx-breach-alerts.herokuapp.com'
+SCAN = 'https://fx-breach-alerts.herokuapp.com/scan'
+TEST_EMAIL_HASH = 'b84c4c03b2af4050ac2d3b105e58adf83fa5af05'
```
Comment (nhnt11) [inline loadtests/loadtest.py]: Want to add a comment to mention that this hash is for test@example.com?

Comment (pdehaan) [reply loadtests/loadtest.py]: Wait, how did you get that hash? I tried this, but get a different hash value:

```sh
$ echo "test@example.com" -n | shasum # 4235978ef5585c7910233bfa7b8aed61e96b3003
```

Comment (pdehaan) [reply loadtests/loadtest.py]: Odd, even <kbd>$ node scripts/make-sha1-hashes.js</kbd> gives me an interestingly different result:

> You entered: [test@example.com], sha1 hash of lowercase: 567159d622ffbb50b11b0efd307be358624a26ee

File: loadtests/loadtest.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from molotov import scenario
+
+
+HOMEPAGE = 'https://fx-breach-alerts.herokuapp.com'
+SCAN = 'https://fx-breach-alerts.herokuapp.com/scan'
+# This hash is for test@example.com
+TEST_EMAIL_HASH = 'b84c4c03b2af4050ac2d3b105e58adf83fa5af05'
```
Comment (nhnt11) [inline loadtests/loadtest.py]: Your way of generating the hash was correct: this hash is not the correct hash of test@example.com. Could you update this please? Thanks for catching this :)

File: loadtests/loadtest.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+from os import getenv
+from molotov import scenario
+
+
+SERVER_URL = getenv('SERVER_URL', 'https://fx-breach-alerts.herokuapp.com')
+HOMEPAGE_URL = SERVER_URL
+SCAN_URL = '{}/scan'.format(SERVER_URL)
+# This hash is for test@example.com (via `node scripts/make-sha1-hashes.js`)
+TEST_EMAIL_HASH = '567159d622ffbb50b11b0efd307be358624a26ee'
```
Comment (pdehaan) [inline loadtests/loadtest.py]: @nhnt11 Added a comment and made the SERVER_URL a bit more configurable via an env variable.

**Comment by pdehaan** [comment]
Early results:

```sh
$ ./run.sh # Running homepage (GET `/`) test only...

WEIGHT_HOMEPAGE=1
WEIGHT_SCAN=0
DURATION=180
PROCESSES=20
WORKERS=10
DELAY=0

**** Molotov v1.6. Happy breaking! ****
Forking 20 processes
...
SUCCESSES: 44999 | FAILURES: 0 | WORKERS: 200
```

~250 RPS for static homepage.

---

```sh
$ ./run.sh # Running /scan results page (POST `/scan`) test only...

WEIGHT_HOMEPAGE=0
WEIGHT_SCAN=1
DURATION=180
PROCESSES=20
WORKERS=10
DELAY=0

**** Molotov v1.6. Happy breaking! ****
Forking 20 processes
...
SUCCESSES: 4023 | FAILURES: 1 | WORKERS: 200
```

~22 RPS for /scan results page (probably due to high breach alerts for test email acct).

**Pull request merged**